Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -2305,23 +2305,23 @@ void RenderGrid::gridAreaPositionForGridItem(const RenderBox& gridItem, GridTrac
         gridAreaPositionForInFlowGridItem(gridItem, direction, start, end);
 }
 
-ContentPosition static resolveContentDistributionFallback(ContentDistribution distribution)
+std::pair<OverflowAlignment, ContentPosition> static resolveContentDistributionFallback(ContentDistribution distribution)
 {
     switch (distribution) {
     case ContentDistribution::SpaceBetween:
-        return ContentPosition::Start;
+        return { OverflowAlignment::Default, ContentPosition::Start };
     case ContentDistribution::SpaceAround:
-        return ContentPosition::Center;
+        return { OverflowAlignment::Safe, ContentPosition::Center };
     case ContentDistribution::SpaceEvenly:
-        return ContentPosition::Center;
+        return { OverflowAlignment::Safe, ContentPosition::Center };
     case ContentDistribution::Stretch:
-        return ContentPosition::Start;
+        return { OverflowAlignment::Default, ContentPosition::Start };
     case ContentDistribution::Default:
-        return ContentPosition::Normal;
+        return { OverflowAlignment::Default, ContentPosition::Normal };
```

Comment:
Since start is functionally safe, is a pair here actually necessary since every possible input value returns?

Also maybe you could just use `StyleContentAlignmentData::isCentered()` instead of having this function?

Reply:
Also, if it makes it easier in the subsequent code, maybe StyleContentAlignmentData can just return safe explicitly for all the ContentDistribution values.

Reply:
Yeah that makes sense but I think I slightly prefer this approach since it makes it much more clear what is going on around the call site and I feel like just using `isCentered` would slightly detract at that. 

In theory if we ever add a new `ContentDistribution` value we would also be forced to adjust this code properly.